docs/usage/targeted.md

```diff
@@ -0,0 +1,281 @@
+---
+order: 1
+---
+
+# nf-core/crisprseq: Usage
+
+## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/crisprseq/usage](https://nf-co.re/crisprseq/usage)
```

Comment:
Will this be shown correctly on the website if you have 2 subpages for `output` and `usage` ?